Running the test, I get:
```
% flatpak run org.freecadweb.FreeCAD -t 0       
FreeCAD 0.21.2, Libs: 0.21.2R33771 (Git)
© Juergen Riegel, Werner Mayer, Yorik van Havre and others 2001-2023
FreeCAD is free and open-source software licensed under the terms of LGPL2+ license.
FreeCAD wouldn't be possible without FreeCAD community.
  #####                 ####  ###   ####  
  #                    #      # #   #   # 
  #     ##  #### ####  #     #   #  #   # 
  ####  # # #  # #  #  #     #####  #   # 
  #     #   #### ####  #    #     # #   # 
  #     #   #    #     #    #     # #   #  ##  ##  ##
  #     #   #### ####   ### #     # ####   ##  ##  ##

Assembly4 workbench (v0.12.4) loaded
Sheet Metal workbench loaded
Gtk-Message: 19:46:10.828: Failed to load module "canberra-gtk-module"
Gtk-Message: 19:46:10.828: Failed to load module "canberra-gtk-module"
(qt.qpa.qgnomeplatform) Could not find color scheme  ""
Qt: Session management error: Could not open network socket
A3 available
import TestFemImport 
import TestObjectExistance 
import TestFemCommon 
import TestObjectCreate 
import TestObjectType 
import TestObjectOpen 
import TestMaterialUnits 
import TestMeshCommon 
import TestMeshEleTetra10 
import TestMeshGroups 
import TestResult 
import TestCcxTools 
import TestSolverCalculix 
import TestSolverElmer 
import TestSolverZ88 
Traceback (most recent call last):
  File "/app/freecad/Mod/Test/qtunittest.py", line 79, in runClicked
    test = unittest.defaultTestLoader.loadTestsFromName(testName)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/unittest/loader.py", line 217, in loadTestsFromName
    test = obj()
           ^^^^^
  File "/app/freecad/Mod/Test/TestApp.py", line 62, in All
    suite.addTest(tryLoadingTest(test))
                  ^^^^^^^^^^^^^^^^^^^^
  File "/app/freecad/Mod/Test/TestApp.py", line 37, in tryLoadingTest
    return unittest.defaultTestLoader.loadTestsFromName(testName)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/unittest/loader.py", line 162, in loadTestsFromName
    module = __import__(module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/lib/python3.11/site-packages/shiboken2/files.dir/shibokensupport/feature.py", line 139, in _import
    return original_import(name, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/freecad/Mod/Path/TestPathApp.py", line 25, in <module>
    from PathTests.TestPathProfile import TestPathProfile
  File "/app/lib/python3.11/site-packages/shiboken2/files.dir/shibokensupport/feature.py", line 139, in _import
    return original_import(name, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/freecad/Mod/Path/PathTests/TestPathProfile.py", line 31, in <module>
    from PathTests.TestPathAdaptive import getGcodeMoves
  File "/app/lib/python3.11/site-packages/shiboken2/files.dir/shibokensupport/feature.py", line 139, in _import
    return original_import(name, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/freecad/Mod/Path/PathTests/TestPathAdaptive.py", line 27, in <module>
    import Path.Op.Adaptive as PathAdaptive
  File "/app/lib/python3.11/site-packages/shiboken2/files.dir/shibokensupport/feature.py", line 139, in _import
    return original_import(name, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/freecad/Mod/Path/Path/Op/Adaptive.py", line 32, in <module>
    import area
  File "/app/lib/python3.11/site-packages/shiboken2/files.dir/shibokensupport/feature.py", line 139, in _import
    return original_import(name, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SystemError: type Boost.Python.enum has the Py_TPFLAGS_HAVE_GC flag but has no traverse function
System exit
```

This doesn't happen with the current runtime on `stable`

--- a/org.freecadweb.FreeCAD.yaml
+++ b/org.freecadweb.FreeCAD.yaml
@@ -176,7 +176,7 @@ modules:
     buildsystem: simple
     build-commands:
       - ./bootstrap.sh --prefix=/app --with-libraries=filesystem,program_options,regex,system,thread,python,date_time,chrono,atomic --with-python=/usr/bin/python3
-      - ./b2 install cxxflags="-I/usr/include/python3.10" -j $FLATPAK_BUILDER_N_JOBS
+      - ./b2 install cxxflags="-I/usr/include/python3.11" -j $FLATPAK_BUILDER_N_JOBS
     sources:
       - type: archive
         url: https://boostorg.jfrog.io/artifactory/main/release/1.80.0/source/boost_1_80_0.tar.bz2

--- a/org.freecadweb.FreeCAD.yaml
+++ b/org.freecadweb.FreeCAD.yaml
@@ -242,8 +242,8 @@ modules:
       - /lib/python3.*/site-packages/numpy/*/tests
     sources:
       - type: archive
-        url: https://files.pythonhosted.org/packages/d0/b0/13e2b50c95bfc1d5ee04925eb5c105726c838f922d0aaddd57b7c8be0f8b/numpy-1.26.3.tar.gz
-        sha256: 697df43e2b6310ecc9d95f05d5ef20eacc09c7c4ecc9da3f235d39e71b7da1e4
+        url: https://files.pythonhosted.org/packages/a4/9b/027bec52c633f6556dba6b722d9a0befb40498b9ceddd29cbe67a45a127c/numpy-1.24.4.tar.gz
+        sha256: 80f5e3a4e498641401868df4208b74581206afbee7cf7b8329daae82676d9463
 
   - name: certifi
     buildsystem: simple

--- a/org.freecadweb.FreeCAD.yaml
+++ b/org.freecadweb.FreeCAD.yaml
@@ -179,8 +179,8 @@ modules:
       - ./b2 install cxxflags="-I/usr/include/python3.11" -j $FLATPAK_BUILDER_N_JOBS
     sources:
       - type: archive
-        url: https://boostorg.jfrog.io/artifactory/main/release/1.80.0/source/boost_1_80_0.tar.bz2
-        sha256: 1e19565d82e43bc59209a168f5ac899d3ba471d55c7610c677d4ccf2c9c500c0
+        url: https://boostorg.jfrog.io/artifactory/main/release/1.81.0/source/boost_1_81_0.tar.bz2
+        sha256: 71feeed900fbccca04a3b4f2f84a7c217186f28a940ed8b7ed4725986baf99fa
 
   - name: coin
     buildsystem: cmake-ninja

